Infrastructure-Scripts/Set-AzResourceTags.ps1

```diff
@@ -0,0 +1,74 @@
+<#
+    .SYNOPSIS
+    Sets Tags on All Resources in a Resource Group to match the Tags set on the Resource Group.
+
+    .DESCRIPTION
+    If The Resource Group exists and has been tagged reads the Tags assigned to the Resource Group and replicates them to all Resources within that Resource Group.
+    If there is a Tag with a different Value it will be overwrittend,
+
+    .PARAMETER ResourceGroupName
+    Name of the Resource Group to be created and/or have tags applied.
+
+    .EXAMPLE
+    Set-ResourceTags -ResourceGroupName "das-at-foobar-rg"
+#>
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$ResourceGroupName
+)
+
+try {
+    $Group = Get-AzResourceGroup -ResourceGroupName $ResourceGroupName
+    if ($null -ne $Group.Tags) {
+        $Resources = Get-AzResource -ResourceGroupName $Group.ResourceGroupName
+        foreach ($R in $Resources) {
```

Comment:
Can this $R changed to be $Resource to be more descriptive?

Reply:
Done